packages/q/qt6ct/package.yml

```diff
@@ -0,0 +1,21 @@
+name       : qt6ct
+version    : 0.9
+release    : 1
+source     :
+    - https://github.com/trialuser02/qt6ct/releases/download/0.9/qt6ct-0.9.tar.xz : 0a9f00db7f9c2003b33fad1715cbf2a483bf50a99079b97bf8e5d0bdbd3392cd
+homepage   : https://github.com/trialuser02/qt6ct
+license    : BSD-2-Clause
+component  : desktop.qt
+summary    : Qt6 Configuration Tool
+description: |
+    This program allows users to configure Qt6 settings (theme, font, icons, etc.) under DE/WM without Qt integration.
+builddeps  :
+    - pkgconfig(Qt6UiTools)
+rundeps    :
+    - qt6-svg
+setup      : |
```

Comment:
Can you add the following above `setup`:
```
clang : yes
optimize : thin-lto
```

I like to keep Qt applications on Clang to match the Qt stack itself.

Reply:
After adding those, it become unable to launch
```
Failed to create wl_display (No such file or directory)
qt.qpa.plugin: Could not load the Qt platform plugin "wayland" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vkkhrdisplay, vnc, xcb, wayland-egl, wayland.

fish: Job 1, 'qt6ct' terminated by signal SIGABRT (Abort)

```